tracpro/settings/base.py

```diff
@@ -271,6 +271,7 @@ def _org_scheduler_task(task_name):
     "Administrators": (
         'orgs.org_home',
         'orgs.org_edit',
+        'orgs.fetch_runs',
```

Comment:
@rowanseymour is this not the way to add it to the permissions? I have been running `migrate` but am not getting any feedback that it's doing anything and it doesn't seem to work. Any other input is welcome!